pages/[id].js

```diff
@@ -68,7 +71,7 @@ export default function PlantDetails() {
   );
 }
 
-const Container = styled.div`
+const Container = styled.main`
```

Comment:
Here's another <main> Element. You have a lot of mains nested in other mains. Your site should always have only one <main> Element.